doc/user_guide/install.rst

```diff
@@ -74,6 +74,47 @@ and ``hyperspy_gui_traitsui``. To install hyperspy without the GUI packages, use
     for more details.
 
 
+Further information
+^^^^^^^^^^^^^^^^^^^
+
+When installing packages, ``conda`` will verify that all requirements of `all`
+packages installed in an environment are met; this can lead to situations where
+a solution for dependencies resolution can't be found or an expected solution
+is found, such as not installing the lastest version of libraries or installing
+undesired version. Usually, it highly depends on what libraries are already
+present in the current environment, which can make resolving the dependencies
+satisfiability problematic. In such situation, possible solutions are:
+
```

Comment:
When installing packages, ``conda`` will verify that the requirements of `all`
packages installed in an environment are met. This can lead to situations where
a solution for all dependencies cannot be resolved or the solution may include
installing old or undesired versions of libraries. The requirements depend on which libraries are already
present in the environment as satisfying their respective dependencies may be problematic. In such situations, possible solutions are: